### ISSUE TITLE
Leader key expansion in :Shortcuts menu should be configurable

### ISSUE THREAD

**Issue by mk12** [sunaku/vim-shortcut#6]
The leader key display behaviour in 4195e886db640b84786d086e9d14b1b5c17e71cf doesn't look very good when your leader is `<Space>`. Everything just gets indented by a space. Perhaps make it configurable with something like `g:shortcut_expand_leader_key`?

**Comment by sunaku** [comment]
Good idea. :+1: As an alternative to the configuration option, I'm wondering if I should just do this by default for all non-printable leader keycodes. :thinking:

**Comment by mk12** [comment]
That sounds like a reasonable default to me.
